### PR TITLE
[TS SDK] Add query to fetch all collections that an account has tokens for

### PR DESCRIPTION
**Pull request by 0xmaayan**
### Description
Add query to fetch all collections that an account has tokens for. supports v1 and/or v2 standards

### CODE REVIEW

File: ecosystem/typescript/sdk/src/tests/e2e/indexer.test.ts [hunk 1]
```diff
@@ -235,6 +235,11 @@ describe("Indexer", () => {
       expect(tokens.current_token_ownerships_v2).toHaveLength(2);
     });
 
+    it("gets account current tokens from a specified token standrd", async () => {
```
Comment (gregnazario) [inline ecosystem/typescript/sdk/src/tests/e2e/indexer.test.ts:238]: `standard` :)

**Review by gregnazario — APPROVED** [review]
It looks fine to me.  Could you get @rtso or @bowenyang007 to review the query?

File: ecosystem/typescript/sdk/src/providers/indexer.ts [hunk 2]
```diff
@@ -457,4 +489,39 @@ export class IndexerClient {
...
+   * @param ownerAddress the account address that owns the tokens
+   * @returns the collection address
+   */
+  async getCollectionsWithOwnedTokens(
+    ownerAddress: MaybeHexString,
+    extraArgs?: {
+      tokenStandard?: TokenStandard;
+      options?: PaginationArgs;
+    },
+  ): Promise<GetCollectionsWithOwnedTokensQuery> {
```
Comment (banool) [inline ecosystem/typescript/sdk/src/providers/indexer.ts:505]: The comment about what this returns and the return type here don't match, could you update that?

**Review by banool — APPROVED** [review]
Query looks correct to me but yeah best to get an indexer expert to double check.

**Review by rtso — APPROVED** [review]
Query looks good to me.

**Pull request merged**